packages/rest-typings/src/v1/omnichannel.ts

```diff
@@ -1355,6 +1355,36 @@ const GETOmnichannelContactsSearchSchema = {
 
 export const isGETOmnichannelContactsSearchProps = ajv.compile<GETOmnichannelContactsSearchProps>(GETOmnichannelContactsSearchSchema);
 
+type GETOmnichannelContactHistoryProps = PaginatedRequest<{
+	contactId: string;
+	source?: string;
+}>;
+
+const GETOmnichannelContactHistorySchema = {
+	type: 'object',
+	properties: {
```

Comment:
Shouldn't we mark all the not-required props as nullable true?

Reply:
I also thought, but Kevin explained [here](https://github.com/RocketChat/Rocket.Chat/pull/33043/files/9e3cbf8ea1e4afb1209fcc081dd4ec16f65cf478#r1767402358) that the `nullable: true` on the prop is only when we accept them as null and not necessary that they're not required props.